indexing-service/src/main/java/io/druid/indexing/overlord/RemoteTaskRunner.java

```diff
@@ -511,7 +511,7 @@ public void shutdown(final String taskId)
             response.getContent()
         );
 
-        if (!response.getStatus().equals(HttpResponseStatus.ACCEPTED)) {
+        if (!response.getStatus().equals(HttpResponseStatus.OK)) {
```

Comment:
Can you invert this so it will never be null?

`!HttpResponseStatus.OK.equals(response.getStatus())`